ui/pages/onboarding-flow/secure-your-wallet/index.scss

```diff
@@ -0,0 +1,47 @@
+.secure-your-wallet {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  max-width: 1000px;
+  max-height: 1300px;
+
+  &__details {
+    max-width: 550px;
+  }
+
+  &__actions {
+    & button {
+      margin: 16px;
+    }
+  }
+
+  &__list {
+    list-style: disc inside;
+
+    & li {
```

Comment:
Do we need the `&` here?  I think it's implied with nesting.

Reply:
Indeed CSS sensei!